packages/core/crates/core-strategy/src/auto_funding.rs

```diff
@@ -72,17 +71,15 @@ impl<Db: HoprCoreEthereumDbActions> SingularStrategy for AutoFundingStrategy<Db>
 
                     let to_stake = channel.balance.add(&self.cfg.funding_amount);
 
-                    let _ = fund_channel(
+                    let rx = fund_channel(
                         self.db.clone(),
                         self.tx_sender.clone(),
                         channel.get_id(),
-                        to_stake.clone(),
-                    )
-                        .await?;
+                        to_stake,
+                    ).await?;
+                    std::mem::drop(rx); // The Receiver is not intentionally awaited here
```

Comment:
But that will prohibit the sender to send as well, the sender will return on `poll_ready`:
`Poll::Ready(Err(SendError)) if the receiver has been dropped`, if not polled and combined with `start_send`, it's undefined behavior.

Reply:
The error returned by the sender sender is ignored.